setup.py

```diff
@@ -5,14 +5,20 @@
 try:
     import fontTools  # noqa
 except:
-    print("*** Warning: defcon requires FontTools, see:")
-    print("    fonttools.sf.net")
+    print("*** Warning: trufont requires FontTools, see:")
```

Comment:
with setuptools you could specify, e.g.,`install_requires = ['fonttools>=3.0']` keyword in the `setup` function, so that it could grab the required packages before attempting to install trufont itself.
but you all know that ;)